bounties/level-2/million-lumen-homepage.md

```diff
@@ -0,0 +1,34 @@
+
+# Million Lumen Homepage
+
+| Hunter | Agent
+| :- | :-
+| 800 XLM | 2 Credits
+
+## Description
+
+### What is this task?
+
+Remember the Million Dollar Homepage? It was an interesting concept, and it's high time we implement it on Stellar!
+The task is to create a grid of million pixels up for sale and the cost of each individual pixel is 1 XLM.
+
+### What are the requirements for the bounty hunter?
+
+The developer should have knowledge of Web Technologies and Stellar SDK.
+
+### What are the deliverables?
+
+*Description* <br>
+  * The developer should provide a link to a repository implementing the new functionality.
+
+*Judging Criteria and Metrics* <br>
+  * MVP - Does the functionality match with what's stated in the proposal? And how similar it is to the original concept?
+  * UX - Is the UI intuitive and user friendly? Does it adhere to design standards for UI elements and interactions?
+  
+*Submission Procedure* <br>
+  * Submit a link to your repository as a comment on the specific issue thread for this bounty.
```

Comment:
We've switched from comments on an Issue to a PR on the bounty doc itself. (see https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/stellar-lab-darkmode.md#judging-criteria-and-metrics)

I do think we're going to want to open an Issue as well but for each person on a specific bounty for tracking progress / payout etc. It's important both to be able to track completed submissions as well as intentions for submissions so would be hunters know who else if anyone is already working on this bounty.

Would love to hear more thoughts around the actual flow for bounty hunting.

Reply:
> https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/stellar-lab-darkmode.md#judging-criteria-and-metrics

https://github.com/tyvdh/stellar-quest-bounties/blob/main/bounties/level-2/%F0%9F%9A%A8-stellar-lab-darkmode.md